script/Ciel/Day 10/10_01_CIEL10_8.txt

```diff
@@ -252,17 +286,18 @@ Forget everyone, and beginning tomorrow―――
 [sha:b9757a9fb04ad97a800153e8d2e588b575617978]{
 -- Page 606, Offset 28763.
 -- 目蓋に浮かぶ光景を振り払う。
-I clear my mind of the vision that made its way behind my eyelids.
+I shake my head to clear away the vision that flashed before my eyes.
 }
 [sha:732452cb86ee336f8ee6ff63815d548e8e5ae07d]{
 -- Page 606, Offset 28764.
 -- それはもう過ぎ去ったもので、俺の我が儘だけで呼び止められるものじゃない。
-She's already gone now. She's not something I can selfishly stop anyway.
+She's already gone.
+ I can't bring her back with nothing but my own selfish thoughts.
```

Comment:
Second seems misinterpreted. He's reflecting about his regret in not stopping her.

>I couldn't have kept her here through just the power of my own selfishness anyway.